build.gradle

```diff
@@ -70,9 +72,11 @@ publishing {
         maven(MavenPublication) {
             // Defaults, for clarity
             groupId = getGroup()
-            artifactId = getName()
+            artifactId = 'aiven-kafka-connect-jdbc'
```

Comment:
getName() is giving me "maven" also, I can pull the project name from settings.gradle, but given that the project name changed, that would be a breaking change for some build tools.

Reply:
Thanks for catching this one

Reply:
I guess the name should be 'jdbc-connector-for-apache-kafka' same as in other places.

Reply:
You are right @AnatolyPopov

Reply:
I'm fine with changing that, however, that would break existing dependencies on 'aiven-kafka-connect-jdbc'.

Reply:
Yes, that's something we know. We would need to "advertise" the new name under the changelog.